core/src/banking_stage/consumer.rs

```diff
@@ -1065,6 +1092,164 @@ mod tests {
         Blockstore::destroy(ledger_path.path()).unwrap();
     }
 
+    #[test]
+    fn test_bank_process_and_record_transactions_in_last_tick() {
+        solana_logger::setup();
+        let GenesisConfigInfo {
+            genesis_config,
+            mint_keypair,
+            ..
+        } = create_slow_genesis_config(10_000);
+        let bank = Bank::new_no_wallclock_throttle_for_tests(&genesis_config).0;
+        let pubkey = solana_sdk::pubkey::new_rand();
+
+        // setup nonce account with a durable nonce different from the current
+        // bank so that it can be advanced in this bank
+        let durable_nonce = nonce::state::DurableNonce::from_blockhash(&Hash::new_unique());
```

Comment:
`DurableNonce` is already imported.  No need to prefix it with a full path.
```suggestion
        let durable_nonce = DurableNonce::from_blockhash(&Hash::new_unique());
```
Or is it a different `DurableNonce`?